### DISCUSSION TITLE
Multiple result sections for the same lot?

### DISCUSSION THREAD

**Discussion by mdewinne** [OP-TED/eForms-SDK#1092] [Q&A] [answered]
We've noticed that it is possible in award notices to create multiple result sections for the same lot, and even with a different, contradictory value in BT-142-LotResult.

In the case we discovered in our system, a user applied this "method" to list the non-winning tenders in a separate result section for the same lot.

Is this intentional or are there any plans to add a validation rule limiting the result section to exactly one per lot?

**Comment by YvesJo** [comment, marked as answer]
Hi,
This is indeed intentional as it is possible to have multiple subsequent LotResults for a given lot (e.g. contracts within a FA, or purchases using a DPS which may need to be reported over a given period).
This shouldn't however be used the way it has been done (i.e. segregating winning and non-winning tenders in 2 different LotResults as this would be confusing.
We could consider adding some rules to prevent this, I'm however not sure this could be done for SDK 1.14, this has to be checked internally.
KR
